Sources/Danger/GitHubDSL.swift

```diff
@@ -17,489 +13,449 @@ public struct GitHub: Decodable {
+        public let htmlUrl: String
     }
-
-    // MARK: - Properties
-
-    /// The number of the pull request.
-    public let number: Int
-
-    /// The title of the pull request.
-    public let title: String
-
-    /// The markdown body message of the pull request.
-    public let body: String?
-
-    /// The user who submitted the pull request.
-    public let user: GitHubUser
-
-    /// The user who is assigned to the pull request.
-    public let assignee: GitHubUser?
-
-    /// The users who are assigned to the pull request.
-    public let assignees: [GitHubUser]?
-
-    /// The ISO8601 date string for when the pull request was created.
-    public let createdAt: Date
-
-    /// The ISO8601 date string for when the pull request was updated.
-    public let updatedAt: Date
-
-    /// The ISO8601 date string for when the pull request was closed.
-    public let closedAt: Date?
-
-    /// The ISO8601 date string for when the pull request was merged.
-    public let mergedAt: Date?
-
-    /// The merge reference for the _other_ repo.
-    public let head: GitHubMergeRef
-
-    /// The merge reference for _this_ repo.
-    public let base: GitHubMergeRef
-
-    /// The state for the pull request: open, closed, locked, merged.
-    public let state: PullRequestState
-
-    /// A boolean indicating if the pull request has been locked to contributors only.
-    public let isLocked: Bool
-
-    /// A boolean indicating if the pull request has been merged.
-    public let isMerged: Bool?
-
-    /// The number of commits in the pull request.
-    public let commitCount: Int?
-
-    /// The number of comments in the pull request.
-    public let commentCount: Int?
-
-    /// The number of review-specific comments in the pull request.
-    public let reviewCommentCount: Int?
-
-    /// The number of added lines in the pull request.
-    public let additions: Int?
-
-    /// The number of deleted lines in the pull request.
-    public let deletions: Int?
-
-    /// The number of files changed in the pull request.
-    public let changedFiles: Int?
-
-    /// The milestone of the pull request
-    public let milestone: GitHubMilestone?
-
-    /// The link back to this PR as user-facing
-    public let htmlUrl: String
 }
 
-// MARK: - GitHubUser
-
-/// A GitHub user account.
-public struct GitHubUser: Decodable, Equatable {
-    // MARK: - CodingKeys
-
-    enum CodingKeys: String, CodingKey {
-        case id
-        case login
-        case userType = "type"
+extension GitHub {
+    /// A GitHub user account.
+    public struct User: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case id
+            case login
+            case userType = "type"
+        }
+
+        public enum UserType: String, Decodable {
+            case user = "User"
+            case organization = "Organization"
+            case bot = "Bot"
+        }
+
+        /// The UUID for the user organization.
+        public let id: Int
```

Comment:
<!--
  0 failure: 
  1 warning:  Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 151;
-->

- :warning: Variable name should be between 3 and 40 characters long: 'id' (`identifier_name`)